zen/scene/scene.c

```diff
@@ -51,6 +51,66 @@ zn_scene_move_board_binding_handler(
   }
 }
 
+static void
+zn_scene_switch_focused_view_binding_handler(
+    uint32_t time_msec, uint32_t key, void* data)
+{
+  struct zn_scene* scene = data;
+  UNUSED(time_msec);
+
+  if (scene->focused_view == NULL) return;
+  // Should be mapped
+  struct zn_board* focused_board = scene->focused_view->board;
+  if (focused_board == NULL) {
+    zn_error("Something went wrong");
+    return;
+  }
+  if (wl_list_length(&focused_board->view_list) == 1) {
+    zn_warn("Only 1 view");
+    return;
+  }
+
+  struct zn_view* new_focused_view;
+  if (key != KEY_TAB) {
+    zn_error("Invalid key: %u", key);
+    return;
+  }
```

Comment:
I prefer to use `zn_assert`.
I think assertion is a good way to indicate a programmatically unreachable situation.

```suggestion
  if (!zn_assert(key == KEY_TAB, "Invalid key: %u", key)) {
    return;
  }
```

The same can be applied to `zn_scene_align_focused_view_binding_handler`